src/js/fable-core/Long.js

```diff
@@ -48,17 +48,22 @@ export default function Long(low, high, unsigned) {
     this.unsigned = !!unsigned;
 }
 
-Long.prototype.Equals = function (other) {
-    return equals(this, other);
-}
-
-Long.prototype.CompareTo = function (other) {
-    return compare(this, other);
-}
-
-Long.prototype.toString = function () {
-    return toString(this);
-}
+Long.prototype.Equals = function (x) { return equals(this, x); }
+Long.prototype.CompareTo = function (x) { return compare(this, x); }
+Long.prototype.toString = function () { return toString(this); }
+Long.prototype.toNumber = function () { return toNumber(this); }
+Long.prototype.neg = function () { return op_UnaryNegation(this); }
+Long.prototype.not = function () { return op_LogicalNot(this); }
+Long.prototype.add = function (x) { return op_Addition(this, x); }
+Long.prototype.sub = function (x) { return op_Subtraction(this, x); }
+Long.prototype.mul = function (x) { return op_Multiply(this, x); }
+Long.prototype.div = function (x) { return op_Division(this, x); }
+Long.prototype.mod = function (x) { return op_Modulus(this, x); }
+Long.prototype.shl = function (x) { return op_LeftShift(this, x); }
+Long.prototype.shr = function (x) { return op_RightShift(this, x); }
+Long.prototype.and = function (x) { return op_BitwiseAnd(this, x); }
+Long.prototype.or = function (x) { return op_BitwiseOr(this, x); }
+Long.prototype.xor = function (x) { return op_ExclusiveOr(this, x); }
```

Comment:
Sorry I didn't notice this in my previous review. I'd prefer to keep most of the methods in `Long` as module functions instead of attaching them to the prototype. I did this intentionally  for Fable 2, also following the new structure for F# classes (most of the members are now module functions instead of instance members, which btw is what happens in F# AST). The reason is to make tree shaking more effective. Any module function that's not used can be eliminated from the final bundle, but if everything is attached to the prototype the whole `Long.js` file will be included as soon as you make any reference to it.